src/middleware/issueDetails.middleware.js

```
@@ -114,9 +114,10 @@ const fetchIssueEntitiesCount = fetchOne({
  * @returns {string}
  */
 const issueErrorMessageHtml = (errorMessage, issue) =>
-    `<p class="govuk-error-message">${errorMessage}</p>${
-      issue ? issue.value ?? '' : ''
-    }`
+    `${
+      issue && issue.value ? issue.value : (issue && issue.length ? issue : '')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)